mp/src/game/server/player_command.cpp

```diff
@@ -425,19 +424,6 @@ void CPlayerMove::RunCommand ( CBasePlayer *player, CUserCmd *ucmd, IMoveHelper
 		}
 	}
 
-	IServerVehicle *pVehicle = player->GetVehicle();
-
-	// Latch in impulse.
-	if ( ucmd->impulse )
-	{
-		// Discard impulse commands unless the vehicle allows them.
-		// FIXME: UsingStandardWeapons seems like a bad filter for this. The flashlight is an impulse command, for example.
-		if ( !pVehicle || player->UsingStandardWeaponsInVehicle() )
-		{
-			player->m_nImpulse = ucmd->impulse;
-		}
-	}
```

Comment:
Bad removal. This was checking if the vehicle didn't exist, so we still need this.